compiler/rustc_target/src/spec/mod.rs

```diff
@@ -2493,6 +2524,12 @@ pub struct TargetOptions {
     /// If not provided, LLVM will infer the float ABI from the target triple (`llvm_target`).
     pub llvm_floatabi: Option<FloatAbi>,
 
+    /// Picks a specific ABI for this target. This is *not* just for "Rust" ABI functions,
+    /// it can also affect "C" ABI functions; the point is that this flag is interpreted by
+    /// rustc and not forwarded to LLVM.
+    /// So far, this is only used on x86.
+    pub rust_abi: Option<RustAbi>,
```

Comment:
Alternative name suggestion: `rustc_abi`

Reply:
Yeah that would be an option. Let's see what other think.